packages/ember-metal/lib/properties.js

```diff
@@ -136,10 +136,8 @@ export function defineProperty(obj, keyName, desc, data, meta) {
         obj[keyName] = data;
       }
     } else {
-      value = desc;
```

Comment:
`value` here is being set for `didDefineProperty a few lines below. Seems like we have no tests, but perhaps it should not be removed?